barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/plookup_tables/README.md

```diff
@@ -0,0 +1,98 @@
+# Lookup Table Structure in Barretenberg
```

Comment:
I reworked this to be based on how 32-bit XOR currently works in the repo. There are so many ways to present this information that I think I could just keep tweaking it forever but I think this gets all of the main points across